makefile

```diff
@@ -17,6 +17,7 @@ supabase-migrate-database: ## apply the migrations to the database
 
 supabase-reset-database: ## reset (and clear!) the database
 	npx supabase db reset
+	$(MAKE) supabase-migrate-database
```

Comment:
I still have doubts this is really needed. 
According the the [docs](https://supabase.com/docs/reference/cli/supabase-db-reset), it should already apply migrations at this stage:

> Recreates the local Postgres container and applies all local migrations found in supabase/migrations directory

Can you double check?

Reply:
This shouldn't be needed. The `supabase db reset` command apply all migrations